modules/apps/transfer/keeper/genesis_test.go

```diff
@@ -48,10 +48,16 @@ func (suite *KeeperTestSuite) TestGenesis() {
 	genesis := suite.chainA.GetSimApp().TransferKeeper.ExportGenesis(suite.chainA.GetContext())
 
 	suite.Require().Equal(types.PortID, genesis.PortId)
-	suite.Require().Equal(traces.Sort(), genesis.DenomTraces)
+	suite.Require().Equal(denomTraces.Sort(), genesis.DenomTraces)
 	suite.Require().Equal(escrows.Sort(), genesis.TotalEscrowed)
 
 	suite.Require().NotPanics(func() {
 		suite.chainA.GetSimApp().TransferKeeper.InitGenesis(suite.chainA.GetContext(), *genesis)
 	})
+
+	ctx := suite.chainA.GetContext()
+	for _, denomTrace := range denomTraces {
+		_, found := suite.chainA.GetSimApp().BankKeeper.GetDenomMetaData(ctx, denomTrace.IBCDenom())
```

Comment:
`ctx` var feels redundant given that we pass `suite.chainA.GetContext()` directly above in many other places

```suggestion
	for _, denomTrace := range denomTraces {
		_, found := suite.chainA.GetSimApp().BankKeeper.GetDenomMetaData(suite.chainA.GetContext(), denomTrace.IBCDenom())
```